ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -380,42 +344,34 @@ private void handlePrintedTopic(
       final String printTopic,
       final SqlBaseParser.PrintTopicContext ignored
   ) {
-    final RestResponse<InputStream> topicResponse =
+    final RestResponse<StreamPublisher<String>> topicResponse =
         makeKsqlRequest(printTopic, restClient::makePrintTopicRequest);
 
     if (topicResponse.isSuccessful()) {
-      try (Scanner topicStreamScanner = new Scanner(topicResponse.getResponse(), UTF_8.name());
-          StatusClosable toClose = terminal.setStatusMessage("Press CTRL-C to interrupt")
```

Comment:
The new `handlePrintedTopic()` logic does not appear to set this status message. Can we add it back?

Reply:
ack